Composer/packages/server/src/models/bot/botProject.ts

```diff
@@ -325,8 +325,7 @@ export class BotProject {
 
   public async deleteAllFiles(): Promise<boolean> {
     try {
-      await this.deleteFilesFromBottomToUp(this.dir);
-      await this.fileStorage.rmDir(this.dir);
+      await this.fileStorage.rmDir(this.dir, true);
```

Comment:
This is where you'd call the new method (which can just be an invocation of `rimraf`; see the other comment).